content/syntax-highlighting.mdz

```diff
@@ -6,8 +6,7 @@
 ---
 # Syntax Highlighting
 
-Mendoza supports syntax highlighting for a few languages
-built it, and also has a simple mechanism for adding more
+Mendoza supports syntax highlighting for a few languages out of the box, and also has a simple mechanism for adding more
```

Comment:
the wording of `supports a few languages built in` was a bit weird. I tried clarifying with `...out of the box`.